include/BBTrack.h

```diff
@@ -108,13 +111,21 @@ protected slots:
 
 protected:
 	void paintEvent( QPaintEvent * );
+	void resizeEvent( QResizeEvent * re )
```

Comment:
Correct me if I'm wrong here (not great with C++), but if each class which inherits `TrackContentObjectView` simply calls `m_needsUpdate = true`' and then it's parent's `resizeEvent(...)`, then it the `m_needsUpdate` flag can probably be happen directly in `TrackContentObjectView::resizeEvent(...)`.  Also I don't think it's a `virtual void` in `SampleTCOView` if it contains code.